src/components/calendar/calendar.js

```diff
@@ -187,6 +187,10 @@ export const BCalendar = Vue.extend({
       type: String
       // default: null
     },
+    tabIndex: {
```

Comment:
I don't like implementation with a `tab-index` prop.
Users could get confused the the `tabindex` attribute.

I would prefer a similar implementation to the `no-key-nav` prop in the `<b-tabs>` componnet.